accumulator/transform.go

```diff
@@ -129,21 +126,35 @@ func swapCollapses(swaps, collapses [][]arrow, forestRows uint8) {
 			if len(collapses[cr]) == 0 {
 				continue
 			}
+
 			mask := swapIfDescendant(rowcol, collapses[cr][0], r, cr, forestRows)
+
 			if mask != 0 {
-				// fmt.Printf("****col %v becomes ", collapses[ch])
+				// fmt.Printf("****col %v becomes ", collapses[cr])
 				collapses[cr][0].to ^= mask
-				// fmt.Printf("%v due to %v\n", collapses[ch], rowcol)
+				// fmt.Printf("%v due to %v\n", collapses[cr], rowcol)
 			}
 		}
 	}
 }
 
+// delRoot deletes all the roots of a given tree
+// TODO would be more elegant not to have this here. But
+// easier to just delete the root first...
+func delRoot(dels []uint64, rootPresent bool, rootPos uint64) ([]uint64, bool) {
```

Comment:
where is this function being called from?

Reply:
Ah this was for something that I was gonna make into a separate function but decided it wasn't worth the extra memory overhead for.

Deleted now